aiokafka/protocol/message.py

```diff
@@ -1,6 +1,10 @@
 import io
 import time
 from binascii import crc32
+from collections.abc import Iterable
```

Comment:
```suggestion
from typing import Iterable
```
It doesn't work in 3.8, which we have to support yet:
```python
>>> from collections.abc import Iterable
>>> Iterable[int]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'ABCMeta' object is not subscriptable
>>> from typing import Iterable
>>> Iterable[int]
typing.Iterable[int]
```